hack/bootstrap-kubestellar.sh

```diff
@@ -0,0 +1,263 @@
+    curl -sL https://github.com/kcp-dev/kcp/releases/latest | grep "</h1>" | head -n 1 | sed -e 's/<[^>]*>//g' | xargs
+}
+
+kubestellar_installed() {
+    if [[ "$(which mailbox-controller)" == "" || "$(which placement-translator)" == "" || "$(which scheduler)" == "" ]]; then
+        echo "false"
+    else
+        echo "true"
+    fi
+}
+
+kubestellar_running() {
+    if [[ "$(pgrep -f mailbox-controller)" == "" ||  "$(pgrep -f placement-translator)" == "" || "$(pgrep -f 'scheduler -v')" == "" ]]; then
+        echo "false"
+    else
+        echo "true"
+    fi
+}
+
+kubestellar_get_latest_version() {
+    curl -sL https://github.com/kcp-dev/edge-mc/releases/latest | grep "</h1>" | head -n 1 | sed -e 's/<[^>]*>//g' | xargs
+}
+
+get_os_type() {
+  case "$OSTYPE" in
+      linux*)   echo "linux" ;;
+      darwin*)  echo "darwin" ;;
+      *)        echo "Unsupported operating system type: $OSTYPE" >&2 ; exit 1 ;;
+  esac
+}
+
+get_arch_type() {
+  case "$HOSTTYPE" in
+      x86_64*)  echo "amd64" ;;
+      aarch64*) echo "arm64" ;;
+      *)        echo "Unsupported architecture type: $HOSTTYPE" >&2 ; exit 1 ;;
+  esac
+}
+
+get_full_path() {
+    echo "$(cd "$1"; pwd)"
+}
+
+kcp_version=""
+kubestellar_version=""
+os_type=""
+arch_type=""
+folder=""
+strip_bin="false"
+verbose=""
+kcp_address=""
+kubestellar_imw=""
+kubestellar_wmw=""
+
+while (( $# > 0 )); do
+    case "$1" in
+    (--kcp-version)
+        if (( $# > 1 ));
+        then { kcp_version="$2"; shift; }
+        else { echo "$0: missing release version" >&2; exit 1; }
+        fi;;
+    (--kubestellar-version)
+        if (( $# > 1 ));
+        then { kubestellar_version="$2"; shift; }
+        else { echo "$0: missing release version" >&2; exit 1; }
+        fi;;
+    (--os)
+        if (( $# > 1 ));
+        then { os_type="$2"; shift; }
+        else { echo "$0: missing OS type" >&2; exit 1; }
+        fi;;
+    (--arch)
+        if (( $# > 1 ));
+        then { arch_type="$2"; shift; }
+        else { echo "$0: missing architecture type" >&2; exit 1; }
+        fi;;
+    (--bind)
+        if (( $# > 1 ));
+        then { kcp_address="$2"; shift; }
+        else { echo "$0: missing ip address" >&2; exit 1; }
+        fi;;
+    (--imw)
+        if (( $# > 1 ));
+        then { kubestellar_imw="$2"; shift; }
+        else { echo "$0: missing workspace name" >&2; exit 1; }
+        fi;;
+    (--wmw)
+        if (( $# > 1 ));
+        then { kubestellar_wmw="$2"; shift; }
+        else { echo "$0: missing workspace name" >&2; exit 1; }
+        fi;;
+    (--folder)
+        if (( $# > 1 ));
+        then { folder="$2"; shift; }
+        else { echo "$0: missing installation folder" >&2; exit 1; }
+        fi;;
+    (--verbose|-V)
+        verbose="-V";;
+    (-h|--help)
+        echo "Usage: $0 [--kcp-version release_version] [--kubestellar-version release_version] [--os linux|darwin] [--arch amd64|arm64] [--folder installation_folder] [--strip-bin] [-V|--verbose]"
```

Comment:
The case statement here does not recognize a `--strip-bin` option.  I think that is good, but it means that this usage message needs to be adjusted.

Reply:
ok, fixed